src/main/java/org/opensearch/flowframework/transport/CreateWorkflowAction.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.transport;
+
+import org.opensearch.action.ActionType;
+
+/**
+ * External Action for public facing RestCreateWorkflowActiom
+ */
+public class CreateWorkflowAction extends ActionType<WorkflowResponse> {
+
+    // TODO : Determine external action prefix for plugin
+    /** The name of this action */
+    public static final String NAME = "workflows/create";
```

Comment:
@dbwiddis Other plugins have an action prefix for each of their transport action names. For example, Anomaly detection has external and internal action prefixes (IE `cluster:admin/opendistro/ad/` + `detector/jobmanagement` for the AnomalyDetectorJobTransportAction). 

Not sure if its necessary for us to also prepend something similar to our transport action names, but if so, I'm open to suggestions on what action names we should use